### PR TITLE
Docs on Passing Additional Properties to OpenAPI Generator

### PR DESCRIPTION
**Pull request by MatKuhr**
## What Has Changed?

title

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [x] You verified all new and changed links still work (changing the `id` or name of a file can break links)
- [x] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.

### CODE REVIEW

File: docs/java/features/rest/generate-rest-client.mdx [hunk 1]
```diff
@@ -194,6 +194,11 @@ As an example refer to the plugin XML below:
 
 ## Available Parameters
 
+:::tip Passing Additional Properties to the Underlying OpenAPI Generator
+
+You can pass arbitrary additional configuration properties to the underlying OpenAPI generator via `<additionalProperties>` or `--additional-properties`.
```
Comment (artemkovalyov) [inline docs/java/features/rest/generate-rest-client.mdx:199]:
```suggestion
You can pass arbitrary additional configuration properties to the underlying open source OpenAPI generator via `<additionalProperties>` or `--additional-properties`.
```

Comment (artemkovalyov) [reply docs/java/features/rest/generate-rest-client.mdx:199]: I would also make "underlying open source OpenAPI generator" a link.

**Review by artemkovalyov — COMMENTED** [review]
Let's mention the underlying generator is open source, and give a link to it.

File: docs/java/features/rest/generate-rest-client.mdx (hunk 1 above)
Comment (MatKuhr) [reply docs/java/features/rest/generate-rest-client.mdx]: Good idea, thanks!

**Review by artemkovalyov — APPROVED** [review]
LGTM

**Pull request merged**